phenopacketlab-restapi/src/main/java/org/monarchinitiative/phenopacketlab/restapi/controller/MetadataController.java

```diff
@@ -0,0 +1,57 @@
+package org.monarchinitiative.phenopacketlab.restapi.controller;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import org.monarchinitiative.phenopacketlab.core.PhenopacketLabMetadata;
+import org.monarchinitiative.phenopacketlab.core.model.Resource;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping(value = "${api.version}/metadata")
+public class MetadataController {
+
+    private final PhenopacketLabMetadata metadataService;
+
+    public MetadataController(PhenopacketLabMetadata metadataService) {
+        this.metadataService = metadataService;
+    }
+
+    @GetMapping("{prefix}")
+    public ResponseEntity<Resource> metadataByPrefix(@PathVariable("prefix") @Parameter(description = """
+                                                       __Accepted:__
```

Comment:
I think the accepted prefixes depend on what the `PhenopacketLabMetadata` offers. I would do this instead:
```java
    private final PhenopacketLabMetadata metadataService;
    private final List<String> acceptedPrefixes;

    public MetadataController(PhenopacketLabMetadata metadataService) {
        this.metadataService = metadataService;
        this.acceptedPrefixes = metadataService.resources()
                .map(Resource::getNamespacePrefix)
                .toList();
    }

...

@GetMapping("{prefix}")
    public ResponseEntity<Resource> metadataByPrefix(@PathVariable("prefix") @Parameter(description = """
WE NEED TO UPDATE THE DESCRIPTION..
""") String prefix) {
        if (!acceptedPrefixes.contains(prefix.toUpperCase())) {
            return ResponseEntity.badRequest().build();
        }
        return ResponseEntity.of(metadataService.resourceByPrefix(prefix));
    }
```